docs/README.md

```diff
@@ -123,3 +130,420 @@ something like
+the command's `handle` method to look like this:
+
+```typescript
+// src/commands/CreatePost.ts::handle
+public handle(register: Register): void {
+  register.events(new PostCreated(this.postId, this.title, this.content, this.author))
+}
+```
+
+#### 4. First entity
+
+Now we have a command and an event, however, we do not have any data representation of a
+`Post`. To do so, we will create an `entity`.
+
+```bash
+boost new:entity Post --fields title:string content:string author:string --reduces PostCreated
+```
+
+This time Booster has created a file called `Post.ts` in the `src/entities` directory.
+An entity is a reduction function applied to a series of events. For this example, we won't
+take into account the previous state of the `Post`, so we will create a new one
+from the Event's data.
+
+```typescript
+// src/entities/Posts.ts
+@Entity
+export class Post {
+  public constructor(public id: UUID, readonly title: string, readonly content: string, readonly author: string) {}
+
+  @Reduces(PostCreated)
+  public static reducePostCreated(event: PostCreated, currentPost?: Post): Post {
+    return new Post(event.postId, event.title, event.content, event.author)
+  }
+}
+```
+
+#### 5. First read model
+
+So far, we have been adding data to our blog. Now, we will retrieve our data using read
+models. A read model allows us to query our entire entities or just a subset of their
+attributes. Let's project our `Post` Entity:
+
+```bash
+boost new:read-model PostReadModel --fields title:string author:string --projects Post:id
+```
+
+As you might guess, the command will generate a file called `PostReadModel.ts` under
+`src/read-models`:
+
+```text
+boosted-blog
+└── src
+    └── read-models
+        └── PostReadModel.ts
+```
+
+There are two things to do when creating a read model:
+
+1. Define who is authorized to query it
+1. and filter out the entity fields that are not needed in the read model
+
+Read models and commands constitute the public API of a Booster application. With the
+`CreatePost` command, we authorized `all` to execute it. This time we will be doing the
+same for the read model. Furthermore, We will exclude the `content` field from the `Post`
+entity so it won't be visible when querying this read-model. To do so, edit the class to
+look like this:
+
+```typescript
+// src/read-models/PostReadModel.ts
+@ReadModel({
+  authorize: 'all'// Specify authorized roles here. Use 'all' to authorize anyone
+})
+export class PostReadModel {
+  public constructor(
+    public id: UUID,
+    readonly title: string,
+    readonly author: string,
+  ) {}
+
+  @Projects(Post, "postId")
+  public static projectPost(entity: Post, currentPostReadModel?: PostReadModel): PostReadModel {
+      return new PostReadModel(entity.id, entity.title, entity.author);
+  }
+
+}
+```
+
+#### 6. Deployment
+
+We have everything we need to deploy our application to the cloud. It is as simple as
+running this command:
+
+```bash
+boost deploy -e production
```

Comment:
what does mean the environment? Are there any differences if I deploy the booster app using development, testing or whatever value in that option?

Reply:
Excellent questions, the answer is I don't know :dumb: but I wouldn't explain the meaning and options in this 10m guide. Those will appear in the deploying section, and surely every provider will also need an explanation of what is the difference in using every environment.

Maybe @alvaroloes or @tainguyenbui can answer this better.

Reply:
Booster allows defining more than one environment, similar to other frameworks like Rails. We introduced this feature when we started working in the local runtime, to be able to run the project locally using the `development` environment or publish it to the cloud using the `production` environment. There are no limitations on how you use this feature, so you could also configure staging environments or have personal environments for the developers using different accounts or parameterizing the app name.